device-server/src/main/kotlin/com/badoo/automation/deviceserver/ios/simulator/Simulator.kt

```diff
@@ -1073,4 +1073,15 @@ class Simulator(
         }
         remote.shell("xcrun simctl spawn $udid launchctl setenv ${envsArguments.joinToString(" ")}")
     }
+
+    override fun getEnvironmentVariable(env: String): String {
+        if (env.isEmpty()) {
+            logger.debug(logMarker, "Passed empty environment variable to fetch for Simulator $this")
+            return ""
+        }
```

Comment:
Let's move check for an empty string in request handler in DeviceServer.kt 
and let's throw error back if string is empty